src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig

```diff
@@ -30,6 +30,14 @@
     <div class="sf-tabs">
         <div class="tab">
             {% set exception_as_array = exception.toarray %}
+            {% set _exceptions_with_user_code = [] %}
+            {% for i, e in exception_as_array %}
+                {% for trace in e.trace %}
+                    {% if (trace.file is not empty) and ('/vendor/' not in trace.file) and not loop.last %}
```

Comment:
In this condition we should also include: `and ('/var/cache/' not in trace.file)` to avoid displaying the compiled container.